environment.yml

```diff
@@ -18,7 +19,7 @@ dependencies:
   - geopy
   - humanfriendly
   - importlib_resources
-  - iris>=3.4.0
+  - iris>=3.6.0
```

Comment:
this may conflict with Tool's current environment (the Py311 env that @remi-kazeroni is using now) - let's see, I had issues with such a pin yesterday

Reply:
hope so - of course, this pin is essential here :grin: